.github/git_rules/comment_responder.py

```
@@ -89,31 +80,76 @@ def respond_to_comment(self):
+            except:
+                files_info += "파일 정보 로드 실패\n"
+            
+            # 모든 코멘트들 (AI 리뷰 결과 포함)
+            comments_info = "모든 코멘트들:\n"
+            try:
+                comments = list(self.pr.get_issue_comments())
+                for comment in comments:
+                    author = comment.user.login
+                    body = comment.body
+                    comments_info += f"- {author}: {body}\n"
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)